Games/Retro_ML.SuperMarioKart/Neural/Scoring/FinishedRaceScoreFactor.cs

```diff
@@ -1,4 +1,5 @@
-﻿using Retro_ML.Game;
```

Comment:
Line 77 should use RankingMult variable instead of ExtraField

https://github.com/Tourmi/Retro-ML/pull/27/files#diff-9e1a1c2bef76f28973a549b702c9fb96e52b88d9f489e1cc57684100a205255eR77